api/well-known/README.md

```diff
@@ -0,0 +1,74 @@
+# Discovery APIs
+The APIs described here allow a user of the veraison to obtain and view meta information about the veraison deployment. Information about deployment can be queried according to the service e.g. (provisioning or verification).
+
+The relevant resource is created using in response to a client `GET`. The queried information is then outputted as shown examples below.
+
+## Well know API: Provisioning service
+Each well-know provisioning session is associated to its own resource, with the
+following attributes:
+
+* The allowed provisioning mediatypes;
+* The version of the provisioning service.
+
+### Quering information about the provisioning service
+```text
+>> Request:
+  GET /.well-known/veraison/provisioning
+  Host: veraison.example
+  Accept: application/vnd.veraison.provisioning+json
+  Content-Type: application/json
+
+
+<< Response:
+  HTTP/1.1 200 OK
+  Content-format: application/vnd.veraison.provisioning-well-known-session+json
```

Comment:
I guess we could just use `application/json` here, or something shorter like `vnd.veraison.provisioning-discovery+json` (using the word "session" in this context is not correct.)